modules/tripleliftBidAdapter.js

```diff
@@ -27,7 +27,8 @@ export const tripleliftAdapterSpec = {
     tlCall = tryAppendQueryString(tlCall, 'v', '$prebid.version$');
 
     if (bidderRequest && bidderRequest.refererInfo) {
-      let referrer = bidderRequest.refererInfo.referer;
+      // TODO: is 'page' the right value here?
```

Comment:
Based on [this](https://github.com/prebid/Prebid.js/issues/5637#issuecomment-1116554262), yes 'page' is correct. We are interested in the actual page where the auction is happening, not the `document.referrer`.